packages/dso-toolkit/src/styles/dso.scss

```diff
@@ -12,7 +12,7 @@
 @import "components/attachments-counter";
 // @import "components/alert"; -> Moved to @dso-toolkit/styling
 @import "components/anchor";
-@import "components/badge";
+// @import "components/badge"; -> Moved to @dso-toolkit/styling
```

Comment:
De file waar deze `@import` naar verwijst mag verwijderd worden

Reply:
Inmiddels is hij weg